common/app/common/pages/ImageData.java

```diff
@@ -0,0 +1,36 @@
+package common.pages;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Image;
+
+public class ImageData extends Base {
+
+    private final Image image;
+
+    private ImageData(final Image image) {
+        this.image = image;
+    }
+
+    public static ImageData of(final Image image) {
+        return new ImageData(image);
+    }
+
+    public String getThumbImage() {
+        return nullIfEmpty(image.getUrl());
+    }
+
+    public String getBigImage() {
+        return nullIfEmpty(image.getUrl());
+    }
+
+    public String getPlaceholderImage() {
+        return "http://placehold.it/350x150";
+    }
+
+    private String nullIfEmpty(final String s) {
```

Comment:
I think StringUtils is just great for this and included in the classpath:

http://commons.apache.org/proper/commons-lang/javadocs/api-3.4/org/apache/commons/lang3/StringUtils.html#stripToNull(java.lang.String)